docs/rfc/rfc-1.adoc

```diff
@@ -43,6 +43,34 @@ collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
 == In Depth
 
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+* The total size of the group
+* The number of signers required to sign messages
+* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong".
```

Comment:
We also need to determine the minimal number of active group members during the key generation needed so that the key generation result can be accepted. See [this](https://github.com/keep-network/tbtc-v2/pull/44#discussion_r700222208) comment.

Reply:
Yup!, So something like: We want a 51-of-100 wallet, and we want to start moving the funds if we ever dip below 70 active operators and we want to make sure we don't even create the wallet in the first place if we don't have 90 of the operators for the original DKG (so the worst we can have is 51-of-90).